reports/api.py

```diff
@@ -178,6 +186,13 @@ class ChildViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
             ),
         )
         .prefetch_related("guardians")
+        .prefetch_related("guardians__user")
```

Comment:
I wonder whether this would be as good with just a `prefetch_related("guardians__user")`. Could the `prefetch_related("guardians")` be redundant then?

Reply:
It seems to not create anymore queries, because there is a `django_assert_max_num_queries` tester in the `reports/tests/test_api.py::test_children_endpoint`.